config.xml

```diff
@@ -250,7 +250,7 @@ sudo tar -C /usr/local -xzf go1.8.3.linux-amd64.tar.gz</initScript>
         </com.microsoft.azure.vmagent.AzureVMAgentTemplate>
       </vmTemplates>
       <deploymentTimeout>1200</deploymentTimeout>
-      <approximateVirtualMachineCount>3</approximateVirtualMachineCount>
+      <approximateVirtualMachineCount>1</approximateVirtualMachineCount>
```

Comment:
I'm going to guess this is a dynamically adjusted number, so nothing to worry about in the PR